src/v2/api/index.md

```diff
@@ -1518,6 +1518,7 @@ All lifecycle hooks automatically have their `this` context bound to the instanc
 
   <!-- shorthand -->
   <img :src="imageSrc">
+  <img v-bind:src="'/storage/userimages/' + item.imagefile">
```

Comment:
Here is shorthand.
If you add the mixed quotes, you should be added the above into example.